iron-collapse.html

```diff
@@ -216,6 +216,9 @@
       // Set the final size.
       if (size === 'auto') {
         this.style[this._dimensionMax] = '';
+      // call _transitionEnd if no transition is performed, styles need to be set
+      } else if (size === startSize) {
+        this._transitionEnd();
       } else {
         this.style[this._dimensionMax] = size;
       }
```

Comment:
Hi @valdrinkoshi ,
I disagree. My solution would call _transitionEnd when we want animations but when it was not called because of a transition from (0px->0px). You're solution would call _transitionEnd in ALL cases eventually. This would break the tests "enableTransition(false) disables animations" and changes the behaviour of the component outside of the corner case "animation of an empty iron-collapse".